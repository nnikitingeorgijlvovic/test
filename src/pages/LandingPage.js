function LandingPage() {
  return (
    <div className="welcome" role="main" aria-label="Welcome page">
      <div className="container">
        <span className="letter letter-1">W</span>
        <span className="letter letter-2">e </span>
        <span className="letter letter-3">l</span>
        <span className="letter letter-4">c</span>
        <span className="letter letter-5">o</span>
        <span className="letter letter-6">m</span>
        <span className="letter letter-7">e</span>
      </div>
    </div>
  );
}

export default LandingPage;
