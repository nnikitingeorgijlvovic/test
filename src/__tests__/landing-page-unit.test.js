import { create } from "react-test-renderer";
import LandingPage from "../pages/LandingPage";

test("LandingPage", () => {
  const tree = create(<LandingPage />);
  expect(tree).toMatchSnapshot();
});
