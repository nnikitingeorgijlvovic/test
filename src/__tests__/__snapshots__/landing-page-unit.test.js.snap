// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage 1`] = `
<div
  className="welcome"
>
  <div
    className="container"
  >
    <span
      className="letter letter-1"
    >
      W
    </span>
    <span
      className="letter letter-2"
    >
      e 
    </span>
    <span
      className="letter letter-3"
    >
      l
    </span>
    <span
      className="letter letter-4"
    >
      c
    </span>
    <span
      className="letter letter-5"
    >
      o
    </span>
    <span
      className="letter letter-6"
    >
      m
    </span>
    <span
      className="letter letter-7"
    >
      e
    </span>
  </div>
</div>
`;
